docs/modules/clients/pages/java.adoc

```diff
@@ -820,12 +864,22 @@ add only one port. If you need to add a group of ports, then use the method `add
 In the declarative configuration, the element `ports` can be used for
 both single and multiple port definitions.
 
-==== Setting Smart Routing
+==== Configure Cluster Routing Mode
 
-Smart routing defines whether the client operation mode is smart or unisocket.
-See <<java-client-operation-modes, Java Client Operation Modes>> to learn about these modes.
+You can configure the cluster routing mode to suit your requirements, as described in <<java-cluster-routing-modes,Java Cluster Routing Modes>>.
 
-The following are example configurations.
+The following examples show the configuration for each cluster routing mode. 
+
+[NOTE]
+==== 
+* If you are using the `smart` or `unisocket` client operation modes, select **5.4** from the version picker above the navigation pane to see the configuration information. The cluster routing mode described below must not be present in your configuration
```

Comment:
Again, do we want to encourage use of a deprecated option in the 5.5 docs?

Reply:
Moved to end and made into a tip